<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>694</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Andale Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LoginForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QWidget {
  background: transparent;
}
#hostnameLabel {
	color: white;
}

#formFrame { 
    background-color:transparent;
}

QComboBox,
QPushButton {
    border: 1px solid silver;
    background-color: rgb(200, 200, 200);
}

QComboBox::drop-down {
    border: none;
    width: 24px;
    text-align: center;
}

QComboBox::down-arrow {
    image: url(:/resources/dropdown.svg);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="formFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>180</width>
        <height>180</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 90px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>160</width>
        <height>141</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="userInput">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Droid Sans</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border:  none;
border-radius: 3px;
background-color: rgba(255, 255, 255, 0.85);
color: rgb(48, 48, 48);
font: 75 12pt &quot;Droid Sans&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Usu√°rio</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordInput">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Droid Sans</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border:  none;
border-radius: 3px;
background-color: rgba(255, 255, 255, 0.85);
color: rgb(48, 48, 48);
font: 75 12pt &quot;Droid Sans&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Senha</string>
       </property>
       <property name="margin" stdset="0">
        <number>0</number>
       </property>
       <property name="indent" stdset="0">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="hostnameLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>471</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Droid Sans</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;Droid Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Hostname</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>140</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Droid Sans</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qt-lightdm-greeter.qrc">:/resources/seta.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>140</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Droid Sans</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="clock">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>440</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:52pt &quot;Droid Sans&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>12:06:47</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="leaveComboBox">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.0);
selection-background-color: rgb(0, 170, 255);
selection-color: rgb(255, 255, 255);
font: 9pt &quot;Droid Sans&quot;;
border: none;
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="qt-lightdm-greeter.qrc">
         <normaloff>:/resources/leaveIcon.svg</normaloff>:/resources/leaveIcon.svg</iconset>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="sessionCombo">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Droid Sans</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.0);
selection-background-color: rgb(0, 170, 255);
selection-color: rgb(255, 255, 255);
font: 9pt &quot;Droid Sans&quot;;
border: none;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qt-lightdm-greeter.qrc"/>
 </resources>
 <connections/>
</ui>
